### PR TITLE
chore: update specification

### PR DESCRIPTION
**Pull request by wumaolinmaoan**
## Release Notes

- **New Features**
  - Introduced properties for managing post-processing effects in the `Composition` class.
  - Added methods for texture management in the `Composition` class.
  - Added a mapping mechanism for various value types in the new `value-getter-map` file.
  - New method `updateAnimatedObject` added to the `ObjectBindingTrack` class.

- **Improvements**
  - Enhanced error handling in the `buildItemTree` method for better data integrity.
  - Updated multiple classes to use specification-defined data types for improved type safety.
  - Improved initialization logic in the `SpriteComponent` and other classes to ensure proper setup with default values.

- **Documentation**
  - Added comments for clarity and maintainability across various classes and methods.
  
- **Chores**
  - Updated dependency version for `@galacean/effects-specification` to `2.1.0-alpha.1`.

### CODE REVIEW

File: packages/effects-core/src/math/value-getters/color-curve.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Color } from '@galacean/effects-math/es/core/color';
 import type { BezierCurve } from './value-getter';
 import { ValueGetter, createValueGetter } from './value-getter';
-import type { BezierValue } from '@galacean/effects-specification';
+import type { spec } from '@galacean/effects-core';
```
Comment (yiiqii) [inline packages/effects-core/src/math/value-getters/color-curve.ts:4]: `import type * as  spec from '@galacean/effects-specification';`

File: packages/effects-core/src/math/value-getters/vector4-curve.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Vector4 } from '@galacean/effects-math/es/core/vector4';
 import type { BezierCurve } from './value-getter';
 import { ValueGetter, createValueGetter } from './value-getter';
-import type { BezierValue } from '@galacean/effects-specification';
+import type { spec } from '@galacean/effects-core';
```
Comment (yiiqii) [inline packages/effects-core/src/math/value-getters/vector4-curve.ts:4]: 同上

File: packages/effects-core/src/plugins/timeline/playable-assets/color-property-playable-asset.ts [hunk 3]
```diff
@@ -2,14 +2,14 @@ import { effectsClass, serialize } from '../../../decorators';
 import type { Playable, PlayableGraph } from '../../cal/playable-graph';
 import { PlayableAsset } from '../../cal/playable-graph';
 import { PropertyClipPlayable } from '../playables';
-import type { ColorCurveData } from '../../../math';
 import { createValueGetter } from '../../../math';
 import type { Color } from '@galacean/effects-math/es/core';
+import { spec } from '@galacean/effects-core';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/playable-assets/color-property-playable-asset.ts:7]: 同上

File: packages/effects-core/src/plugins/timeline/playable-assets/float-property-playable-asset.ts [hunk 4]
```diff
@@ -4,8 +4,9 @@ import type { Playable, PlayableGraph } from '../../cal/playable-graph';
 import { PlayableAsset } from '../../cal/playable-graph';
 import { PropertyClipPlayable } from '../playables';
 import { createValueGetter } from '../../../math';
+import { spec } from '@galacean/effects-core';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/playable-assets/float-property-playable-asset.ts:7]: 同上

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/math/value-getters/color-curve.ts (hunk 1 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/math/value-getters/color-curve.ts]: 已修改

File: packages/effects-core/src/plugins/timeline/tracks/vector4-property-track.ts [hunk 5]
```diff
@@ -1,9 +1,10 @@
+import { spec } from '@galacean/effects-core';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/tracks/vector4-property-track.ts:1]: 同上

File: packages/effects-core/src/math/value-getters/value-getter-map.ts [hunk 6]
```diff
@@ -0,0 +1,93 @@
+import { spec } from '@galacean/effects-core';
```
[the remark this answers is not in the document]
Comment (wumaolinmaoan) [reply packages/effects-core/src/math/value-getters/value-getter-map.ts]: 已修改

**Review by yiiqii — APPROVED** [review]

**Pull request merged**